src/controller/CHIPDeviceController.h

```diff
@@ -603,6 +618,7 @@ class DLL_EXPORT DeviceCommissioner : public DeviceController, public SessionEst
     Callback::Callback<DefaultFailureCallback> mOnRootCertFailureCallback;
 
     PASESession mPairingSession;
+    PingClient mPingClient;
```

Comment:
I understand DeviceCommisioner as a "extension to DeviceController that allows commisioning functions, i.e. allows for PASE establishment".

By that definition, PingClient has no place here. 
I was trying to read through the spec and generally could not find "echo" or "ping" so have a hard time placing it. If this is specifically for test events to validate the example apps, this should be part of examples. However if the spec does not mandate it, I would like to understand why this is mandatory for tests and how does it fit in - if this is not a spec requirement, why is it part of the certification?